tests/test_pandas.py

```diff
@@ -189,13 +203,26 @@ def test_concat_args() -> None:
         ),
         pd.DataFrame,
     )
+    check(
+        assert_type(
+            pd.concat([df, df2], keys=["df1", "df2"], names=None), pd.DataFrame
+        ),
+        pd.DataFrame,
+    )
     check(
         assert_type(
             pd.concat([df, df2], keys=["df1", "df2"], names=[pd.Timedelta(1, "D")]),
             pd.DataFrame,
         ),
         pd.DataFrame,
     )
+    check(
+        assert_type(
+            pd.concat([df, df2], keys=["df1", "df2"], names=None),
+            pd.DataFrame,
+        ),
+        pd.DataFrame,
+    )
```

Comment:
Unless I'm missing something, I think this test is the same as the test on lines 206-211, so you can delete it.